code/modules/projectiles/guns/energy.dm

```diff
@@ -12,13 +11,16 @@
 	var/projectile_type = /obj/item/projectile/energy/laser // /obj/item/projectile/beam/practice
 	var/modifystate
 	var/charge_meter = 1	//if set, the icon state will be chosen based on the current charge
+	var/unload_sound 	= 'sound/weapons/guns/interact/pistol_magout.ogg'
+	var/reload_sound 	= 'sound/weapons/guns/interact/pistol_magin.ogg'
```

Comment:
These don't seem to be used anywhere. However considering that projectile weapons use unload and reload sounds as well, might be a better idea to move those vars to the top level.